salt/client/spm.py

```diff
@@ -4,8 +4,9 @@
 
 .. versionadded:: Beryllium
 '''
+from __future__ import absolute_import, print_function
```

Comment:
Futures are also python imports, any specific reasons to move them outside the python imports comment header?